api/system-openssh/update

```diff
@@ -42,19 +42,16 @@ foreach my $prop (qw(
 }
 
 my $AllowGroups = '';
-my $obj = $data->{'props'}{'AllowGroups'};
-if ((defined $obj) && ($obj ne "")) {
-  foreach my $g (keys $data->{'props'}{'AllowGroups'} ) {
-      if ($data->{'props'}{'AllowGroups'}{$g} eq 'ssh') {
-          $AllowGroups .= "$g,";
-      } elsif ($data->{'props'}{'AllowGroups'}{$g} eq 'sftp') {
-          $AllowGroups .= "$g:sftp,";
-      }
-  }
-  chop $AllowGroups;
-  $cdb->set_prop('sshd', 'AllowGroups', $AllowGroups);
+my $obj = $data->{'props'}{'AllowGroups'} || {};
+foreach my $g (keys $obj) {
+    if ($obj->{$g} eq 'ssh') {
+        $AllowGroups .= "$g,";
+    } elsif ($obj->{$g} eq 'sftp') {
+        $AllowGroups .= "$g:sftp,";
+    }
 }
-
+chop $AllowGroups;
+$cdb->set_prop('sshd', 'AllowGroups', $AllowGroups) if(ref($data->{'props'}{'AllowGroups'}) eq 'HASH');
```

Comment:
```suggestion
$cdb->set_prop('sshd', 'AllowGroups', $AllowGroups) if(ref($obj) eq 'HASH');
```

Reply:
it is merged, do you want to open a new PR ?

Reply:
I already pushed the commit to master :innocent:

Reply:
super